reference/configuration/framework.rst

```diff
@@ -428,9 +428,11 @@ performance a bit:
             $framework->enabledLocales(['en', 'es']);
         };
 
-If some user makes requests with a locale not included in this option, the
-application won't display any error because Symfony will display contents using
-the fallback locale.
```

Comment:
If there is no special locale parameter in request, this case is applied right ?

I think we should document it. I'm not sure what IS "fallback local", is it the default locale ?

Reply:
I don't understand your comment ... according to the code, if you use `enabled_locales`, the `_locale` will be set unconditionally:

https://github.com/symfony/symfony/blob/184597db05288c68c9eb11dff55239768268f646/src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php#L1083-L1086